ArduPlane/quadplane.cpp

```diff
@@ -2735,7 +2753,7 @@ void QuadPlane::vtol_position_controller(void)
         // call attitude controller
         set_pilot_yaw_rate_time_constant();
         attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(plane.nav_roll_cd,
-                                                                      plane.nav_pitch_cd,
+                                                                      MAX(plane.nav_pitch_cd, q_fwd_nav_pitch_lim_cd),
```

Comment:
should be on 2747

Reply:
I've reworked it to move the angle limiting and forward thrust calculation to just after we get the demanded pitch angle from the position controller so that a consistent pitch angle demand is guaranteed for all downstream processing.